src/pages/workspaces/workspace/WorkspaceContainer.ts

```diff
@@ -379,14 +367,43 @@ const useAppPolling = (workspace) => {
   return { apps, refreshApps };
 };
 
-export const wrapWorkspace =
-  ({ breadcrumbs, activeTab, title, topBarContent, showTabBar = true }) =>
-  (WrappedComponent) => {
+interface WrapWorkspaceProps {
+  breadcrumbs: (props: { name: string; namespace: string }) => ReactNode;
+  activeTab?: string;
+  title: string;
+}
+
+interface WrappedComponentProps extends ComponentPropsWithRef<any> {
+  workspace: Workspace;
+  refreshWorkspace: () => void;
+  analysesData: AppDetails & CloudEnvironmentDetails;
+  storageDetails: StorageDetails;
+}
+
+type WrappedWorkspaceComponent<T extends WrappedComponentProps> = (props: T) => ReactNode;
+
+type WorkspaceWrapperFunction<T extends WrappedComponentProps> = (
+  component: WrappedWorkspaceComponent<T>
+) => WrappedWorkspaceComponent<T>;
+// type WorkspaceWrapperComponent<T extends WrappedComponentProps> = (props: WrapWorkspaceProps) => WorkspaceWrapperComponent<T>
```

Comment:
Is this left over code that can be deleted?

Reply:
it is